breathecode/events/views.py

```diff
@@ -379,6 +379,17 @@ def delete(self, request, format=None, academy_id=None):
         return Response(None, status=status.HTTP_204_NO_CONTENT)
 
 
+# list eventbride webhook
+class OrganizationWebhookView(APIView):
+    @capable_of('read_organization')
+    def get(self, request, academy_id=None):
+
+        webhooks = EventbriteWebhook.objects.filter(organization_id=organization_id)
```

Comment:
organization_id variable does not exist, you need to first retrieve the organization to be able to use it:
```
org = Organization.objects.filter(academy__id=academy_id).first()
if org is None:
    raise ValidationException(f'Academy has no organization', code=400, slug='organization-no-found') 

webhooks = EventbriteWebhook.objects.filter(organization_id=org.id)
```